server/src/test/java/org/opensearch/monitor/fs/FsHealthServiceTests.java

```diff
@@ -182,6 +184,60 @@ public void testLoggingOnHungIO() throws Exception {
         }
     }
 
+    public void testFailsHealthOnHungIOBeyondHealthyTimeout() throws Exception {
+        long healthyTimeoutThreshold = randomLongBetween(500, 1000);
+        long refreshInterval = randomLongBetween(50, 100);
```

Comment:
Should we also add a test with ``refreshInterval`` having larger value than the ``healthyTimeoutThreshold``, as with the current implementation, I believe it will always result into unhealthy threshold breach.